lib/game.rb

```diff
@@ -0,0 +1,102 @@
+require "pry"
+
+class Game
+  attr_accessor :board, :player_1, :player_2, :timer
+
+  WIN_COMBINATIONS = [
+  [0, 1, 2],
+  [3, 4, 5],
+  [6, 7, 8],
+  [0, 3, 6],
+  [1, 4, 7],
+  [2, 5, 8],
+  [0, 4, 8],
+  [6, 4, 2]
+  ]
+
+  def initialize(player_1 = Players::Human.new("X"), player_2 = Players::Human.new("O"), board = Board.new, wargame = false)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+    @wargame = wargame
+    @timer = 1.5
+  end
+
+  def current_player
+    board.turn_count.even?  ? player_1 : player_2
+  end
+
+  def won?
+    WIN_COMBINATIONS.find do |combo|
+      board.cells[combo[0]] == board.cells[combo[1]] && board.cells[combo[1]] == board.cells[combo[2]] && board.cells[combo[0]] != " "
+    end
+  end
+
+  def draw?
+    board.full? && !won?
+  end
+
+  def over?
+    !!won? || draw?
+  end
+
+  def winner
+    board.cells[won?[0]] if won?
+  end
+
+  def turn
```

Comment:
We would prefer to see the CLI logic separated from the game logic in its own class (e.g. TicTacToeCLI). Splitting the interface from the implementation of the game allows for multiple interfaces (e.g. a web app) without requiring changes to the game code.